src/Libraries/PythonNodeModels/Properties/Resources.resx

```diff
@@ -192,7 +192,7 @@
     <value>Use this dropdown to choose the python version/engine to execute your code.</value>
   </data>
   <data name="PythonScriptEditorMigrationAssistantButtonTooltip" xml:space="preserve">
-    <value>Migration Assistant to update your Python 2 scripts to Python 3.</value>
+    <value>Migration Assistant to update your Python 2 scripts to Python 3. Engine must be set to Python 2 version to work.</value>
```

Comment:
@Amoursol @aparajit-pratap Any more opinion on the messaging?

Reply:
Looks good to me.